public/manifest.json

```diff
@@ -1,29 +1,14 @@
 {
-  "name": "WebMark",
+  "name": "__MSG_extensionName__",
   "version": "0.1.0",
-  "description": "Load a random page from your bookmarks!",
+  "description": "__MSG_extensionDescription__",
```

Comment:
so we need to wrap the variables with `__` ?

Reply:
> In manifest.json and CSS files, refer to a string named messagename like this:
> `__MSG_messagename__`
> In your extension or app's JavaScript code, refer to a string named messagename like this:
> `chrome.i18n.getMessage("messagename")`